File: ReverseDigit/ReverseDigit.cs
// Reverse a Number:

using System;
class Program
{
    static void Main()
    {
        int n, temp, reverse = 0;
        Console.Write("Enter number: ");
        n = int.Parse(Console.ReadLine());
        temp = n;
        while (n > 0)
        {
            reverse = reverse * 10 + n % 10;
            n /= 10;
        }
        Console.WriteLine($"Number: {temp} Reversed Number: {reverse}");
    }
}

File: ReverseDigit/ReverseDigit.c
// Reverse a Number:

#include <stdio.h>
int main()
{
    int n, reverse = 0, temp;
    printf("Enter number: ");
    scanf("%d", &n);
    temp = n;
    while (n > 0)
    {
        reverse = reverse * 10 + n % 10;
        n /= 10;
    }
    printf("number: %d Reversed Number: %d", temp, reverse);
}